libs/sdk-ui-dashboard/src/presentation/dashboard/types.ts

```diff
@@ -578,6 +578,29 @@ export interface IDashboardEventing {
     onStateChange?: (state: DashboardState, dispatch: DashboardDispatch) => void;
 }
 
+/**
+ * @internal
+ */
+export enum DashboardPartIds {
```

Comment:
Enums should be named in singular. It works better, i.e., see the usage: `Record<DashboardPartIds, INavigationConfigItem>`. It would read better when it would be a singular.

Reply:
What would you say about naming it `KeyboardNavigationTarget`?

Reply:
Singular ok. My idea was, that these identifiers do not have to serve only as a navigation target, but to identify dashboard part in general